ext/proj4_c_impl/errors.c

```diff
@@ -0,0 +1,31 @@
+#ifndef RGEO_PROJ4_ERRORS_INCLUDED
+#define RGEO_PROJ4_ERRORS_INCLUDED
+
+#include <ruby.h>
+
+#include "preface.h"
+
+#ifdef RGEO_PROJ4_SUPPORTED
+
+#include "errors.h"
+
+RGEO_BEGIN_C
+
+VALUE error_module;
+VALUE rgeo_error;
+VALUE rgeo_invalid_projection_error;
```

Comment:
As I made the experience while working on the last rgeo core patch, I think the `rgeo_error` name can easily be ambiguous in a global scope. I think we could match ruby’s convention, using a `rb_e` prefix and a PascalCase name. In the core repo, I’ve written it `rb_eRGeoError`. Note that this rename only happened in. the v3-dev branch, hence if you want to discuss that further and change namings a bit, I’m definitely ok with patching the core repo :)